### PR TITLE
Feature/register cell

### PR DESCRIPTION
**Pull request by AlekseyPleshkov**
[UIKit] Add UITableView+RegisterCell
[UIKit] Add UITableView+ReusableCell
[UIKit] Add UICollectionView+RegisterCell
[UIKit] Add UICollectionView+ReusableCell

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,14 @@ In process
 ##### UIViewController
 * [Child](https://github.com/sqrline/HoneyKit/blob/master/HoneyKit/UIKit/UIViewController/UIViewController%2BChild.swift). Handles (adds, removes and switches) child of the current view controller.
 
+##### UITableView
```
Comment (BastienBoy) [inline README.md:34]: UITableView should be above UIViewController

Comment (BastienBoy) [inline README.md:34]: UITableView should be above UIViewController

File: README.md [hunk 2]
```diff
@@ -30,6 +30,14 @@ In process
...
+* [RegisterCell](https://github.com/sqrline/HoneyKit/blob/master/HoneyKit/UIKit/UITableView/UITableView%2BRegisterCell.swift). Registers a cell by class or nib for use in creating new table cells.
+* [ReusableCell](https://github.com/sqrline/HoneyKit/blob/master/HoneyKit/UIKit/UITableView/UITableView%2BReusableCell.swift). Returns a reusable table view cell object by class.
+
+##### UICollectionView
```
Comment (BastienBoy) [inline README.md]: Should be below UIButton

**Review by BastienBoy — CHANGES_REQUESTED** [review]
Sorting problem in readme

File: README.md (hunk 2 above)
Comment (AlekseyPleshkov) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (AlekseyPleshkov) [reply README.md:34]: Done

**Comment by AlekseyPleshkov** [comment]
Fixed sorting in readme is done

**Pull request merged**